.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,16 @@
+name: Pull Request
+
+on:
+  pull_request:
+
+jobs:
+  buildAndPackage:
```

Comment:
Hate to nit but we should rename it to just `build` if we removed packaging step.

Reply:
Good catch